### ISSUE TITLE
about the testing and training

### ISSUE THREAD

**Issue by lxh990620** [kxhit/SG_PR#13]
Thanks for your great contribution!
I have a question for testing and training part

You described "We use 1-fold cross-validation and
each sequence is considered as a fold, that is, consider one
sequence as a test set and the others as training sets"

And I understood the LoopClosureDetection based 00,02,05,07,08 sequnce

Could you please tell me in details about which sequnce you used for the 1fold cross validation for testing and training?

Hoping for your reply.
Thanks a lot.

**Comment by kxhit** [comment]
Thanks for your interest!

For example, assume we have 6 sequences (00, 02, 05, 06, 07, 08) in total for the KITTI dataset. Now we can choose 02 as testing data and the others (00, 05, 06, 07, 08) as training data. Basically, each sequence can be chosen as testing data while the remaining other sequences as training data.

Hope it is clear and please feel free to post your confusion.

**Comment by lxh990620** [comment]
Thanka a lot!
You guys are so kind and brilliant
浙大牛逼
